sources/packages/web/src/assets/css/base.css

```diff
@@ -1,15 +1,14 @@
-html * {
-  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
-  font-style: normal;
-  font-weight: 400;
-  line-height: 20px;
-  letter-spacing: 0px;
-}
-
-.bold-text {
-  font-style: normal;
-  font-weight: 700;
-  line-height: 36px;
-  letter-spacing: -0.2px;
-  color: #2965c5;
-}
+html * {

```

Comment:
Why we changed this? is this part of your Story?

Reply:
Not quite part of this, but it was not part of any other specific one when it was introduced 😄 
This is just an effort here and there to try to bring some styles from Figma.
In this PR I added the formio-shared.css as an idea to try to start to centralized the styles used along with the form definitions in only one place instead of configuring one by one, as shown below. (@ann-aot FYI)
![image](https://user-images.githubusercontent.com/61259237/128773483-4fefd46e-1603-475b-8033-909a27e777f6.png)